### PR TITLE
Replacing fvtk by the new viz API

### PR DESCRIPTION
**Pull request by skoudoro**
The goal of this PR is to replace `fvtk` by the new `viz` API in all examples.  Most of the work is done.

Last work to do:
 
- [x]  create `tensor_slicer` in actor instead of using `fvtk.tensor`
- [x] replace `fvtk.contour` by something else. Maybe by #1165 from @kesshijordan in some case.

### CODE REVIEW

**Comment by skoudoro** [comment]
Waiting for Travis but this PR is ready to go.

Anyone for reviewing it? @dmreagan @ranveeraggarwal?

File: dipy/viz/actor.py [hunk 1]
```diff
@@ -877,8 +886,196 @@ def _makeNd(array, ndim):
...
+                  norm=True, opacity=1., scalar_colors=None):
+    """ Slice many tensors as ellipsoids in native or world coordinates
+
+    Parameters
+    ----------
+    evals : (3,) or (X, 3) or (X, Y, 3) or (X, Y, Z, 3) ndarray
+        eigenvalues
+    evecs : (3, 3) or (X, 3, 3) or (X, Y, 3, 3) or (X, Y, Z, 3, 3) ndarray
+        eigenvectors
+    affine : array-
```
Comment (Garyfallidis) [inline dipy/viz/actor.py]: Please remove -

**Comment by Garyfallidis** [comment]
This PR although having many files seems would be easy to merge. However, the patch coverage needs a lift.:)

**Comment by Garyfallidis** [comment]
Great. Thank you @skoudoro !

**Pull request merged**